### PR TITLE
Get TW info on extension load

### PR DESCRIPTION
**Pull request by Thiteago**
Created function to load TW info when extension is opened, also change name of function updateWorkedHours to keep it different from another  function  with same name

### CODE REVIEW

File: assets/scripts/background.js [hunk 1]
```diff
@@ -20,6 +20,10 @@ async function createOffscreen() {
 
 chrome.runtime.onStartup.addListener(createOffscreen);
 chrome.runtime.onInstalled.addListener(createOffscreen);
+chrome.runtime.onConnect.addListener(function(port) {
```
Comment (ErnaneJ) [inline assets/scripts/background.js:23]: Just to follow the design pattern we can change this statement to an arrow function.

```js
chrome.runtime.onConnect.addListener(port =>{
  chrome.offscreen.closeDocument();
  createOffscreen();
});
```

File: assets/scripts/popup.js [hunk 2]
```diff
@@ -1,3 +1,5 @@
+var port = chrome.runtime.connect({name: "extensionOpen"});
```
Comment (ErnaneJ) [inline assets/scripts/popup.js:1]: As the context here is interestingly complex, we can avoid some future problems by checking the possibility of changing the `port` variable declaration to `let` or `const`. By declaring it as `var` we run the 'risk' of making it public in a possible context different from the one we are working with. Do we really want this?

**Review by ErnaneJ — CHANGES_REQUESTED** [review]
Interesting proposal. I observed some points of attention that we need to have. After these adjustments, we can proceed with the PR.

**Comment by Thiteago** [comment]
Changed function to arrow function and variable type

**Review by ErnaneJ — APPROVED** [review]

**Pull request merged**